sdk/src/main/java/com/amplitude/experiment/DefaultExperimentClient.kt

```diff
@@ -107,6 +109,74 @@ internal class DefaultExperimentClient internal constructor(
             UserSessionExposureTracker(it)
         }
 
+    private val isRunningLock = Any()
+    private var isRunning = false
+
+    /**
+     * Start the SDK by getting flag configurations from the server and fetching
+     * variants for the user. The future returned by this function resolves when
+     * local flag configurations have been updated, and the fetch()
+     * result has been received (if the request was made).
+     *
+     * This function determines whether to fetch() based on the result of
+     * the flag configurations cached locally or received in the initial flag
+     * configuration response.
+     *
+     * To explicitly force this request to fetch or not, set the
+     * fetchOnStart configuration option when initializing the SDK.
+     *
+     * Finally, this function will start polling for flag configurations at a
+     * fixed interval. To disable polling, set the pollOnStart
+     * configuration option to `false` on initialization.
+     *
+     * @param user The user to set in the SDK.
+     * @see fetch
+     * @see variant
+     */
+    fun start(user: ExperimentUser?): Future<ExperimentClient>? {
```

Comment:
Add this to the ExperimentClient interface and override in the class implementation